spt/vgui/vgui_utils.hpp

```diff
@@ -6,9 +6,13 @@
 struct AudioState_t;
 #endif
 
-#include "inputsystem\buttoncode.h"
-
+#ifdef BMS
+class IClientMode;
```

Comment:
If it's not needed anywhere else maybe just leave `class IClientMode;` instead of `iclientmode.h` unconditionally? Just add a comment that it's needed for BMS.

Reply:
ok fixed, looks like the interface isnt actually used directly in any of the gui code